src/Serilog/Formatting/Display/OutputProperties.cs

```diff
@@ -74,7 +77,7 @@ public static class OutputProperties
         /// <param name="logEvent">The log event.</param>
         /// <param name="outputTemplate">The output template.</param>
         /// <returns>A dictionary with properties representing the log event.</returns>
-        public static IReadOnlyDictionary<string, LogEventPropertyValue> GetOutputProperties(LogEvent logEvent, MessageTemplate outputTemplate)
+        internal static IReadOnlyDictionary<string, LogEventPropertyValue> GetOutputProperties(LogEvent logEvent, MessageTemplate outputTemplate)
```

Comment:
I think it would make sense to list it prominently. Super surprised if it breaks anything out there today, though.